docs/source/developers.rst

```diff
@@ -0,0 +1,5 @@
+.. _developers:
+.. include:: ../../CONTRIBUTING.rst
```

Comment:
I did not find this file in this PR, however, there is a sentence 
'as ASAP as possible' which is weird.

Reply:
Grepped all three repo, but can't find it.  Which page did you see it on?

Reply:
Ah, got it.